__tests__/api/login/route.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
What's up with this comment?

Reply:
Ah, we talked about it off-line.

Reply:
This logic will be moved out in a follow up PR